tests/QueryTest.php

```diff
@@ -154,6 +156,43 @@ public function testSelect(): void
         $this->assertNull($user->age);
     }
 
+    public function testWhereNot(): void
+    {
+        // implicit equality operator
+        $users = User::whereNot('title', 'admin')->get();
+        $this->assertCount(6, $users);
```

Comment:
This test is failing if I run it on version 3.9.3: the generated query was `{ title: 'admin' }`, which is wrong because it ignore the negation.
In the current master, the query is `{ $not: { title: 'admin' } }`, which is invalid and rejected by the server.
With this patch, the working query is `{ $nor: [ { title: 'admin' } ] }`.

Reply:
>  reviewing the generated query was { title: 'admin' }, which is wrong because it ignore the negation.

I'm not sure if I realized this when originally reviewing [PHPORM-49](https://jira.mongodb.org/browse/PHPORM-49), but the fact that earlier versions completely ignored negations with no feedback to the user sounds like pretty disastrous bug.